recipes/palikiss/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "2.3.1" %}
+{% set packagename = "palikiss" %}
+{% set sha256 = "d9ab68329e48709cffbf5ed775f905bdddebab470d70836bb522993b971d48af" %}
+
+package:
+  name: {{ packagename }}
+  version: 1.1.0
+
+source:
+  sha256: {{ sha256 }}
+  # the repository "fold-grammars" contains sources for several programs, i.e. pKiss, RNAshapes, ...
+  # thus, the version number does not correspond to the {{ packagename }} version number
+  url: https://github.com/jlab/fold-grammars/archive/{{ version }}.tar.gz
+
+build:
+  number: 0
+  skip: False # [osx]
```

Comment:
Do you really mean `false` here ? Usually the value is `True`, i.e. skip the build on OSX

Reply:
yes, this is intentionally. I had to invest some time to make compilation work on OSX. Thus, I've started early versions of my software with skipping OSX and nowadays am able to also support OSX (thanks to @corneliusroemer for assistance with mismatching libc). Should I drop OSX in future versions, I thus know directly how to do that, instead of going through conda readmes again.

Reply:
It is confusing. Please either add a comment or remove it!

Reply:
I decided to remove